### PR TITLE
ON-40097 # Added function to create and update drafts

### PR DESCRIPTION
**Pull request by mymattcarroll**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
-    "@aws-sdk/client-s3": "^3.554.0",
-    "@aws-sdk/lib-storage": "^3.554.0",
+    "@aws-sdk/client-s3": "^3.569.0",
+    "@aws-sdk/lib-storage": "^3.569.0",
     "browser-or-node": "^3.0.0"
   },
   "devDependencies": {
     "@microsoft/eslint-plugin-sdl": "^0.2.2",
-    "@oneblink/types": "github:oneblink/types",
+    "@oneblink/types": "github:oneblink/types#ON-40056",
```
Comment (divporter) [inline package.json:16]: update

File: package.json [hunk 2]
```diff
@@ -62,8 +62,8 @@
     "release": "oneblink-release repository --no-name",
     "start": "tsc-watch",
     "test": "jest",
-    "types": "npm i -D github:oneblink/types",
+    "types": "npm i -D github:oneblink/types#ON-40056",
```
Comment (divporter) [inline package.json:65]: updata

File: src/uploadToS3.ts [hunk 3]
```diff
@@ -86,20 +86,16 @@ async function uploadToS3<T>({
...
     leavePartsOnError: true,
     params: {
       Bucket: bucket,
       Key: key,
       Body: body,
       ContentType: contentType,
-      ServerSideEncryption: 'AES256',
-      Expires: new Date(new Date().setFullYear(new Date().getFullYear() + 1)), // Max 1 year
-      CacheControl: 'max-age=31536000', // Max 1 year(365 days),
-      ACL: 'bucket-owner-full-control',
```
Comment (divporter) [inline src/uploadToS3.ts:102]: why does this go?

Comment (mymattcarroll) [reply src/uploadToS3.ts:102]: These are being set server side in Lambda@Edge. We don't need them here, we just never removed them in the original release.

**Pull request merged**